_test_server/lib/src/handlers/example/http/cross_origin_credentials_handlers.dart

```diff
@@ -28,9 +28,9 @@ Map<String, Handler> exampleHttpCrossOriginCredentialsRoutes = {
 };
 
 String? session;
```

Comment:
Yeah we can do the last one you suggested. I originally tried making it late as well, but the other function in this file may be called prior to session being set and will read it, so that doesn't work.